src/shared/components/home/instances.tsx

```diff
@@ -148,9 +149,15 @@ export class Instances extends Component<any, InstancesState> {
             {items.map(i => (
               <tr key={i.domain}>
                 <td>
-                  <a href={`https://${i.domain}`} rel={relTags}>
-                    {i.domain}
-                  </a>
+                  {link === true || link === undefined ? (
+                    <a href={`https://${i.domain}`} rel={relTags}>
+                      {" "}
+                      {i.domain}{" "}
```

Comment:
Why the spaces before and after?

Reply:
Because the prettier required to be run before committing does that. Pre-pretty I had it as 

```typescript
 <a href={`https://${i.domain}`} rel={relTags}> {i.domain} </a>
```
on one line. The spaces specific to that are to help readability on the web page. The `yarn run v1.22.19` on `git commit` marked it as 

```typescript

    {" "}
    {i.domain}{" "}
```